test/routes/graphql/Query/allUsers.test.ts

```
@@ -0,0 +1,409 @@
+      expect(result.data?.allUsers.edges.length).toBeGreaterThan(0);
+      assertToBeNonNullish(result.data?.allUsers.edges[0]);
+      expect(result.data.allUsers.edges[0].node.name).toBe(uniqueName);
+    });
+
+    test("returns empty result for non-matching name search", async () => {
+      const result = await mercuriusClient.query(Query_allUsers, {
+        headers: {
+          authorization: `bearer ${adminAuthToken}`,
+        },
+        variables: {
+          name: "NonExistentUserName" + faker.string.alphanumeric(10),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)